### PR TITLE
Styling:Milestone 4

### PR DESCRIPTION
**Pull request by Tumworobere**
**Under this milestone,i a chieved the following:**

-  Used the information detailed in Zeplin to style the website.
-  Displayed a list of books.
- Added a book.
- Removed a selected book.
- Live Link : https://tender-hoover-c5014f.netlify.app/

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+  padding: 1rem 5rem 0;
+  background-color: #fff;
   display: flex;
+  align-items: center;
   justify-content: space-between;
-  padding: 0 5rem;
+}
+
+.nav {
+  display: flex;
```
Comment (enaburekhan) [inline src/App.css]: - [ ] Nice job with the styling. In order to ensure we follow the DRY rule in our CSS. I will suggest you declare the highlighted property once in your CSS. Currently, it's occurring about 9x in the CSS. You can achieve this in different ways. One basic way I will show below:

At the top of your css, you can declare the flex property like so:
```
:root {
   --variable-name: flex;
}
```
And then reuse the ` --variable-name` inside the classes like so:
```
.nav {
  display: var(--flex);
 ----other properties---
  ---------
}
```

File: src/App.css [hunk 2]
```diff
@@ -18,19 +43,225 @@ h1 {
...
+}
+
+.circle {
+  border: 1px solid #e8e8e8;
+  height: 2.8rem;
+  width: 2.8rem;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (enaburekhan) [inline src/App.css:66]: - [ ] It will be nice to also adhere to the DRY principle with this highlighted property.

**Review by enaburekhan — CHANGES_REQUESTED** [review]
Hi Tumworobere, 

Good job so far!

- [ ] Major highlights

- The PR summary and commit messages are very descriptive.
- The readme is also very descriptive.
- Passed linters checks.

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by er-contreras — APPROVED** [review]
Hi @Tumworobere, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did most of the work, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Tumworobere** [comment]
> Hi @Tumworobere,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
>     * I see that you did most of the work, well done!
> 
> 
> Felix
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks @eri8-9  for the review!

**Pull request merged**